articles/configure-your-template-strings.mdx

```diff
@@ -0,0 +1,186 @@
+* `date_long` gets a long date format (e.g. 7 June 2022)
+* `date_full` gets a longer date format (e.g. Tuesday, 7 June 2022)
+* `time` gets the time (e.g. 1:00pm)
+* `time_short` gets a short time format (e.g. 1:00 pm)
+* `time_medium` gets a medium time format (e.g. 1:00:00 pm)
+* `time_long` gets a long time format (e.g. 1:00:00 pm NZDT)
+* `time_full` gets a longer time format (e.g. 1:00:00 pm New Zealand Daylight Time)
+
+<comp.Notice info_type="info">
+  All the time and date filters format the date using the site's timezone and the user's locale.
+</comp.Notice>
+
+### Filters with parameters
+
+For some filters, you can provide extra parameters for more flexibility. Add extra parameters by including a `=` character after the filter name.
+
+* `truncate` removes and extra characters beyond a specified number. For example, `{title|truncate=10}` will resolve to the first 10 characters of a message.
+* `default` allows you to provide a fallback value, in case the data is empty
+* `if` resolves to the data, if the parameter is "truthy"
+* `unless` resolves to the data, if the parameter is "falsy"
+
+### Filtering examples
+
+Let's go through some examples for the following file:
+
+<comp.CodeBlock language="yaml" source="short-message.mdx">
+```
+date: 2020-01-02
+empty_value:
+message: "Hello World"
+draft: false
+```
+</comp.CodeBlock>
+
+In the above file we have four keys containing different data types: `date`, `empty_value`, `message`, and `draft`.
+
+Here are some examples of how we can transform this data using different filters:
+
+<comp.CodeBlock language="annotated" source="">
+```
+{date|day}-{date|month}-{date|year} {date|time} ({date|timezone}) #1
+// 01-02-2020 00:00am (Etc/UTC)
+
+{message|uppercase|truncate=5} #2
+// HELLO
+
+{message|if=message} #3
+// Hello World
+
+{message|unless=empty_value} #4
+// Hello World
+
+{empty_value|default="Nothing here!"} #5
+// Nothing here!
+
+{"Unpublished"|if=draft|default="Published"} #6
+// Published
+```
+<comp.Annotation number="1">
+  We can use multiple date filters to control the output string. This template string uses dynamic data placeholders and literal text characters, such as `-` and `()`.
+</comp.Annotation> <comp.Annotation number="2">
+  We can convert text strings to `uppercase` and `truncate` the number of characters to five.
+</comp.Annotation> <comp.Annotation number="3">
+  We can print the value of a key if a value exists using the `if` filter.
+</comp.Annotation> <comp.Annotation number="4">
+  We can print the value of a key based on the value of another key using the `unless` filter. In this case, we print the value of `message` because `empty_key` is falsy.
+</comp.Annotation> <comp.Annotation number="5">
+  We can create a fallback option when a placeholder resolves to an empty result using the `default` filter. In this case, "Nothing here!" is printed because `empty_key` contains no value.
+</comp.Annotation> <comp.Annotation number="6">
```

Comment:
```suggestion
</comp.Annotation> <comp.Annotation number="4">
  We can print the value of a key based on the value of another key using the `unless` filter. In this case, we print the value of `message` because `empty_value` is falsy.
</comp.Annotation> <comp.Annotation number="5">
  We can create a fallback option when a placeholder resolves to an empty result using the `default` filter. In this case, "Nothing here!" is printed because `empty_value` contains no value.
</comp.Annotation> <comp.Annotation number="6">
```